tests/test_server.py

```diff
@@ -48,6 +48,29 @@ def test_drawing_route(self):
         response = self.call_route('/docs/US7654321B2/drawings/1')
         self.assertEqual(200, response.status_code)
 
+    def test_delete_route(self):
+        """ Can Delete File
+        """
+        route = '/docs/US7654321A2'
+        pn = route.split('/')[-1] + '.json'
+        data = b'{"test_key": "test_value"}'
+        path = str((TEST_DIR / 'test-dir/patents' / pn).resolve())
+        with open(path, 'wb') as f:
+            f.write(data)
+
+        is_file = os.path.exists(path) #should exist
+        self.assertTrue(is_file)
+
+        
+        url = f'{PROTOCOL}://{HOST}:{PORT}' + route
+        response = requests.delete(url)
```

Comment:
Parikshit, can you please update the `call_route` function to also send HTTP `delete` requests by passing in another `method` argument, which takes the default value of `get`?